src/adaptors/ease.org/index.js

```diff
@@ -0,0 +1,84 @@
+const utils = require('../utils');
+const Web3 = require('web3');
+const sdk = require('@defillama/sdk')
+const superagent = require('superagent');
+require('dotenv').config({ path: './config.env' });
+const bribePotAbi = require('./bribePotAbi.json');
+const web3 = new Web3("https://rpc.ankr.com/eth");
+
+const bribePotAddress = '0xEA5EdeF17C9be57228389962ba50b98397f1E28C';
+const easeTokenAddress = '0xEa5eDef1287AfDF9Eb8A46f9773AbFc10820c61c';
+
+const gvEase = async () => {
+  const bribePotContract = new web3.eth.Contract(
+    bribePotAbi,
+    bribePotAddress
+  );
+
+  const totalSupply = await bribePotContract.methods.totalSupply().call();
+
+  const genesis = await bribePotContract.methods.genesis().call();
+  const currentTimeStamp = (await web3.eth.getBlock('latest')).timestamp;
+  const currentWeek = Math.floor(((currentTimeStamp - genesis) / (60*60*24*7)));
+  const nextWeek = currentWeek + 1;
+
+  // 212 is the slot for the bribeRates mapping
+  const mappingKey = web3.utils.soliditySha3({type: 'uint256', value: nextWeek}, {type: 'uint256', value: 212});
+  const bribeRates = await web3.eth.getStorageAt(bribePotAddress, mappingKey);
+  // splice StartAmt and ExpireAmt from bribeRates response
+  const rateStartAmtHex = web3.utils.padLeft(bribeRates.slice(0,34), 64);
+  const rateStartAmt = web3.eth.abi.decodeParameter('uint256', rateStartAmtHex);
+  const rateExpireAmtHex = web3.utils.padLeft('0x' + bribeRates.slice(34,67), 64);
+  const rateExpireAmt = web3.eth.abi.decodeParameter('uint256', rateExpireAmtHex);
+
+  // calc how many bribes will be distributed next week
+  const bribePerCurrWeek = await bribePotContract.methods.bribePerWeek().call(); 
```

Comment:
this call is failing for me

Reply:
@dominikpre hey sir, have u managed to fix this issue?

Reply:
Sorry, I missed the first message. Works on my end now. The bribe pot's now distributing to the venal pot. Can you rerun the test?